### PR TITLE
v5.31 ESR upgrade notes

### PR DESCRIPTION
**Pull request by amyblais**
v5.25 ESR support is ending on April. Also planning a forum post, blog post and emails. Also mobile app changelog should be updated.

### CODE REVIEW

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -2,10 +2,10 @@ Important Upgrade Notes
...
-   
+   Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its lifecycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
+
 .. important::
-   Support for Mattermost Server v5.19 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ has come to the end of its lifecycle. Upgrading to v5.25 or later is required.
+   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. From v5.26 Mattermost officially supports PostgreSQL version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
```
Comment (cwarnermm) [inline source/administration/important-upgrade-notes.rst:8]:
```suggestion
   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. From v5.26 Mattermost officially supports PostgreSQL version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
```

File: source/administration/important-upgrade-notes.rst [hunk 2]
```diff
@@ -2,10 +2,10 @@ Important Upgrade Notes
...
 
 .. important::
    TLS versions 1.0 and 1.1 have been deprecated by browser vendors. Starting in v5.31 (January 16, 2021) mmctl will return an error when connected to Mattermost servers deployed with those TLS versions. System Admins will need to explicitly add a flag in their commands to continue to use them. We recommend upgrading to TLS version 1.2 or higher.
```
Comment (cwarnermm) [inline source/administration/important-upgrade-notes.rst:11]:
```suggestion
   TLS versions 1.0 and 1.1 have been deprecated by browser vendors. Starting in v5.31 (January 16, 2021) `mmctl` will return an error when connected to Mattermost servers deployed with those TLS versions. System Admins will need to explicitly add a flag in their commands to continue to use them. We recommend upgrading to TLS version 1.2 or higher.
```

File: source/administration/release-lifecycle.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@ What is a Release Lifecycle?
 During each monthly release, Mattermost backports high severity or high impact security fixes to the previous three monthly releases. `Extended Support Releases <https://docs.mattermost.com/administration/extended-support-release.html>`_ receive backports for security fixes and major bug fixes for the length of their life cycle.  Once a release life cycle has ended, it will no longer receive the security or major bug backports. We follow the same release cycle pattern for all upcoming releases.
 
 .. important::
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its lifecycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (cwarnermm) [inline source/administration/release-lifecycle.rst:9]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```

File: source/administration/upgrade.rst [hunk 4]
```diff
@@ -5,7 +5,7 @@ In most cases you can upgrade Mattermost Server in a few minutes, but the upgrad
 
 .. important::
 
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its lifecycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (cwarnermm) [inline source/administration/upgrade.rst:8]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```

File: source/administration/version-archive.rst [hunk 5]
```diff
@@ -4,7 +4,7 @@ Version Archive
 If you want to check that the version of Mattermost you are installing is the official, unmodified version, compare the SHA-265 checksum or the file's GPG signature with the one published in this version archive. To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
 
 .. important::
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its lifecycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (cwarnermm) [inline source/administration/version-archive.rst:7]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```

**Review by cwarnermm — APPROVED** [review]
Minor proposed changes provided inline.

**Review by esethna — APPROVED** [review]

**Pull request merged**